test/index_test.rb

```diff
@@ -138,11 +138,13 @@ def test_raises_when_writing_invalid_entries
 
   def test_can_write_index
```

Comment:
This was an interesting test failure. What was happening here is it accidentally working. The procedure adds two new index entries. Originally the OIDs are identical. Then, when checking for the order of insertion we sorted by the OID. The stability of this sort depends on too many factors, and was in fact in a different order on Windows. Quick fix is create the second index entry with an OID higher than the first.